src/pages/GapsPatternsPage.tsx

```diff
@@ -56,15 +56,23 @@ const CustomTooltip = ({ active, payload }: TooltipProps<number, string>) => {
 
 function GapsByHour({ lineRef, operatorRef, fromDate, toDate }: BusLineStatisticsProps) {
   const [sortingMode, setSortingMode] = useState<'hour' | 'severity'>('hour')
-
+  const [isLoadingData, setIsLoadingData] = useState(true)
   const hourlyData = useGapsList(fromDate, toDate, operatorRef, lineRef, sortingMode)
-
+  useEffect(() => {
+    console.log('isLoading', isLoadingData)
```

Comment:
better to remove this console statement

Reply:
Haha debugging left overs, sorry I'll fix it ;)